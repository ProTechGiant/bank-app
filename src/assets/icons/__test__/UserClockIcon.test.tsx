import { create } from "react-test-renderer";

import { UserClockIcon } from "../UserClockIcon";

describe("icon/UserClockIcon", () => {
  it("renders a UserClockIcon", () => {
    const result = create(<UserClockIcon />);

    expect(result).toMatchSnapshot();
  });
});
