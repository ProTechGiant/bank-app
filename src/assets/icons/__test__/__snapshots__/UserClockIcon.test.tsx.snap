// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/UserClockIcon renders a UserClockIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="19"
  bbWidth="18"
  fill="none"
  focusable={false}
  height={19}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 19,
        "width": 18,
      },
    ]
  }
  vbHeight={18}
  vbWidth={19}
  width={18}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M11.5 7.29688C13.2266 7.29688 14.6719 5.76562 14.6719 3.80469C14.6719 1.89062 13.2188 0.40625 11.5 0.40625C9.78125 0.40625 8.32812 1.91406 8.32812 3.82031C8.33594 5.76562 9.77344 7.29688 11.5 7.29688ZM11.5 8.67188C10.2578 8.67188 9.15625 8.9375 8.24219 9.36719C9.35938 10.3203 10.0703 11.7266 10.0703 13.2969C10.0703 13.7422 10.0156 14.1719 9.89844 14.5859H16.3359C17.6641 14.5859 18.1172 14.1641 18.1172 13.4297C18.1172 11.4219 15.5469 8.67188 11.5 8.67188ZM4.86719 17.3594C7.07812 17.3594 8.92969 15.5156 8.92969 13.2969C8.92969 11.0781 7.09375 9.24219 4.86719 9.24219C2.64844 9.24219 0.8125 11.0781 0.8125 13.2969C0.8125 15.5312 2.64844 17.3594 4.86719 17.3594ZM3.00781 14C2.67188 14 2.41406 13.7422 2.41406 13.4141C2.41406 13.0859 2.67969 12.8281 3.00781 12.8281H4.32812V11.0781C4.32812 10.7578 4.59375 10.5 4.92188 10.5C5.25 10.5 5.51562 10.7578 5.51562 11.0781V13.4141C5.51562 13.7422 5.25781 14 4.92188 14H3.00781Z"
      fill={
        {
          "payload": 4294134835,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
